Explorer/Assets/DCL/PluginSystem/Global/ExplorePanelPlugin.cs

```diff
@@ -135,16 +137,49 @@ public override void Dispose()
                 navmapController.InitialiseAssetsAsync(assetsProvisioner, ct).Forget();
                 backpackInitialization.Invoke(ref builder, arguments);
 
-                mvcManager.RegisterController(new ExplorePanelController(viewFactoryMethod, navmapController, settingsController, backpackSubPlugin.backpackController!,
+                mvcManager.RegisterController(new ExplorePanelController(viewFactoryMethod, navmapController, settingsController, backpackSubPlugin.backpackController!, arguments.PlayerEntity, builder.World,
                     new ProfileWidgetController(() => explorePanelView.ProfileWidget, web3IdentityCache, profileRepository, webRequestController),
-                    new SystemMenuController(() => explorePanelView.SystemMenu, builder.World, arguments.PlayerEntity, webBrowser, web3Authenticator, userInAppInitializationFlow)));
+                    new SystemMenuController(() => explorePanelView.SystemMenu, builder.World, arguments.PlayerEntity, webBrowser, web3Authenticator, userInAppInitializationFlow),
+                    dclInput));
+
+                explorePanelOpener = new PersistentExplorePanelOpenerController(
+                    PersistentExplorePanelOpenerController.CreateLazily(exploreOpener, null), mvcManager);
 
-                mvcManager.RegisterController(new PersistentExplorePanelOpenerController(
-                    PersistentExplorePanelOpenerController.CreateLazily(exploreOpener, null), mvcManager)
+                mvcManager.RegisterController(explorePanelOpener
                 );
+
+                RegisterHotkeys();
             };
         }
 
+        private void RegisterHotkeys()
+        {
+            dclInput.Shortcuts.MainMenu.performed += OnMainMenuHotkeyPressed;
+            dclInput.Shortcuts.Map.performed += OnMapHotkeyPressed;
+            dclInput.Shortcuts.Settings.performed += OnSettingsHotkeyPressed;
+            dclInput.Shortcuts.Backpack.performed += OnBackpackHotkeyPressed;
+        }
+
+        private void OnMainMenuHotkeyPressed(InputAction.CallbackContext obj)

```

Comment:
I propose to move bindings to a separate class - it's not responsibility of the plugin